app/src/main/java/com/xmartlabs/scasas/criminalintent/CrimeFragment.java

```diff
@@ -0,0 +1,42 @@
+package com.xmartlabs.scasas.criminalintent;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+
+public class CrimeFragment extends Fragment {
+  private Crime crime;
+  private EditText titleField;
+
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    crime = new Crime();
```

Comment:
Yup. Due to the lifecycle, you don't need to override this method only to initialize a variable. The next lifecycle method is `onCreateView`.